builtin/game/item.lua

```diff
@@ -155,6 +155,24 @@ function core.yaw_to_dir(yaw)
 	return {x = -math.sin(yaw), y = 0, z = math.cos(yaw)}
 end
 
+function core.is_colored_paramtype(ptype)
+	return (ptype == "color") or (ptype == "colorfacedir") or
+		(ptype == "colorwallmounted")
+end
+
+function core.strip_param2_color(param2, paramtype2)
+	if (not core.is_colored_paramtype(paramtype2)) then
+		return nil
+	end
+	if (paramtype2 == "colorfacedir") then
```

Comment:
I don't want to be too pedantic but here are a way too many parenthesis. http://dev.minetest.net/Lua_code_style_guidelines#Misc point 4